programs/library-bridge/LibraryBridgeHandlers.cpp

```diff
@@ -399,4 +410,202 @@ void ExternalDictionaryLibraryBridgeExistsHandler::handleRequest(HTTPServerReque
 }
 
 
+CatBoostLibraryBridgeRequestHandler::CatBoostLibraryBridgeRequestHandler(
```

Comment:
"Request" and "ping" handlers for catboost in library-bridge. Sort of similar to the external dictionary handlers. More details are mentioned in the commit message.